doc/release_notes/v3.0.0.rst

```diff
@@ -62,6 +62,8 @@ Changes
   ``pyjpegls`` or ``pylibjpeg`` with ``pylibjpeg-libjpeg`` can be used instead (:issue:`2008`).
 * Using Pillow with JPEG 2000 encoded > 8-bit multi-sample data (such as RGB) now raises an
   exception as Pillow cannot decode such data correctly (:issue:`2006`)
+* Added an exception if an :class:`~numpy.ndarray` is used to set *Pixel Data*
```

Comment:
Raise an exception? I was parsing this as "make an exception for something".

Reply:
Done